movies/src/test/java/comp.fundacionjala/movies/CustomerTest.java

```diff
@@ -0,0 +1,61 @@
+package comp.fundacionjala.movies;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class CustomerTest {
+
+    @Test
+    public void aCustomerInstanceIsNotNull() {
+        Customer customer=new Customer("Juan");
+        assertNotNull(customer);
+    }
+    @Test
+    public void aCustomerInstanceCanBeCreatedGivenACustomerName(){
```

Comment:
Please Give format to all your classes
Take advantage from your IDE
Press ctrl + shift + alt + L